admin/class-wp-security-bp-admin.php

```diff
@@ -213,34 +194,30 @@ public function check_all(){
 	 * @since    1.0.0
 	 */
 	public function run_ajax_calls() {
-
 		if ( wp_doing_ajax() ) {
-			
+
 			$action = empty( $_POST['action'] ) ? '' : $_POST['action'];
```

Comment:
![nonce_verify](https://user-images.githubusercontent.com/43264089/53844070-e58e3780-3fa4-11e9-997b-5779f22defdd.JPG)

Reply:
nonce verification is in my to do list, still have to choose how to pass it to JS